emcc.py

```diff
@@ -2592,7 +2589,7 @@ def check_memory_setting(setting):
   else:
     # wasm includes the mem init in the wasm binary. The exception is
     # wasm2js, which behaves more like js.
-    options.memory_init_file = True
+    # TODO(sbc): Error out here in --memory-init-file used.
     settings.MEM_INIT_IN_WASM = True
```

Comment:
If we don't have support anymore for a memory init file when using wasm, I think `settings.MEM_INIT_IN_WASM` can go away?

Reply:
Yes that is that followup: #18868 :)